<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlBarWidget</class>
 <widget class="QWidget" name="ControlBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_Background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/backgrounds/bg_green_day.png);</string>
   </property>
   <widget class="QPushButton" name="pushButton_Home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/transparency.png);
border-image: url(:/images/transparency.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="homescreen.qrc">
      <normaloff>:/icons/home_day.png</normaloff>:/icons/home_day.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Settings">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/transparency.png);
border-image: url(:/images/transparency.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="homescreen.qrc">
      <normaloff>:/icons/settings_day.png</normaloff>:/icons/settings_day.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="homescreen.qrc"/>
 </resources>
 <connections/>
</ui>
